src/ulfius.c

```diff
@@ -585,7 +585,7 @@ int mhd_iterate_post_data (void * coninfo_cls, enum MHD_ValueKind kind, const ch
   
   struct connection_info_struct * con_info = coninfo_cls;
   size_t cur_size = size;
-  char * data_dup = nstrndup(data, size); // Force value to end with a NULL character
```

Comment:
Ah, that explains it. I started getting a missing reference after I updated the repository.